### ISSUE TITLE
How would this work with arbundles?

### ISSUE THREAD

**Issue by 21e8** [th8ta/ArConnect#55]
Hi,

just came across this and wanted to integrate it with my site.
However I am using `arbundles` and they require each `dataItem` to sign with an `ArweaveSigner`.
How would this translate, if possible, with ArConnect?

**Comment by martonlederer** [comment]
Sadly we don't support it yet, an option would be using the `signature` function maybe?

**Comment by 21e8** [comment]
@martonlederer I am not sure I would call that function as it expects an `UInt8Array` yet the dataItems have their own class `DataItem` which exposes its own `sign` function which is what I am currently calling

**Comment by martonlederer** [comment]
Right... Well this I'm not sure what ArBundles needs exactly, but if we added support for the functionality that ArBundles needs, they would need to integrate that too

**Comment by 21e8** [comment]
I raised an issue with them and will close this one as soon as there is a solution:

https://github.com/Bundlr-Network/arbundles/issues/24

**Comment by martonlederer** [comment]
Solved in 1.0
https://docs.arconnect.io/api/sign-dataitem
https://docs.arconnect.io/api/dispatch
